op-deployer/pkg/deployer/standard/standard.go

```diff
@@ -335,6 +339,27 @@ func ArtifactsHashForTag(tag string) (common.Hash, error) {
 	}
 }
 
+// DefaultHardforkScheduleForTag is used to determine which hardforks should be activated by default given a
+// contract tag. For example, passing in v1.6.0 will return all hardforks up to and including Granite. This allows
+// OP Deployer to set sane defaults for hardforks. This is not an ideal solution, but it will have to work until we get
+// to MCP L2.
+func DefaultHardforkScheduleForTag(tag string) *genesis.UpgradeScheduleDeployConfig {
+	sched := &genesis.UpgradeScheduleDeployConfig{
+		L2GenesisRegolithTimeOffset: op_service.U64UtilPtr(0),
+		L2GenesisCanyonTimeOffset:   op_service.U64UtilPtr(0),
+		L2GenesisDeltaTimeOffset:    op_service.U64UtilPtr(0),
+		L2GenesisEcotoneTimeOffset:  op_service.U64UtilPtr(0),
+		L2GenesisFjordTimeOffset:    op_service.U64UtilPtr(0),
+		L2GenesisGraniteTimeOffset:  op_service.U64UtilPtr(0),
+	}
+
+	if tag == "op-contracts/v1.8.0-rc.4" {
+		sched.ActivateForkAtGenesis(rollup.Holocene)
+	}
+
+	return sched
+}
+
```

Comment:
Should this return an error if an unrecognized tag string is passed in? Maybe using a "tag" type instead of "string" type for the arg would make it easier?

Reply:
This is a good callout. It needs to return a default for non-tagged releases too, so this should really activate holocene unless the tag is v17.0 or below. I'll update this.